### PR TITLE
[MM-33648] - Remove possible confusion point after a member notifies the admin about inviting more users the "Not right now" button remains

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR adds a new screen that comes after the notification process to the admin has ended.
#### Ticket Link
[MM-33648](https://mattermost.atlassian.net/browse/MM-33648?atlOrigin=eyJpIjoiOGI5YTdjYTc5NGY5NGEzOTg1ZjgwNzk1NzZmZjA1OTEiLCJwIjoiaiJ9)

#### Screenshots

https://user-images.githubusercontent.com/28563179/111295092-0db4d480-865c-11eb-8e1c-4020901cc3e1.mov

### CODE REVIEW

File: components/confirm_notify_admin_modal/confirm_notify_admin_modal.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+           }
+       }
+       padding: 0;
+       width: 100%;
+   }
+
+   .modal-centered {
+       text-align: center;
+   }
+}
```
Comment (pablovelezvidal) [inline components/confirm_notify_admin_modal/confirm_notify_admin_modal.scss:130]: missing end of lines

**Review by pablovelezvidal — APPROVED** [review]
Great work @AGMETEOR .

**Review by nickmisasi — APPROVED** [review]
Small question, otherwise LGTM. We should get design to look at this before merge, and if possible to test in a spinwick, Steve too.

File: components/user_limit_modal/user_limit_modal.tsx [hunk 2]
```diff
@@ -10,34 +10,47 @@ import {trackEvent, pageVisited} from 'actions/telemetry_actions';
...
         );
     }, []);
 
+    useEffect(() => {
+        if (notificationProcessDone === true) {
+            props.actions.closeModal(ModalIdentifiers.UPGRADE_CLOUD_ACCOUNT);
+            props.actions.openModal({
+                modalId: ModalIdentifiers.CONFIRM_NOTIFY_ADMIN,
+                dialogType: ConfirmNotifyAdminModal,
+            });
```
[the remark this answers is not in the document]
Comment (AGMETEOR) [reply components/user_limit_modal/user_limit_modal.tsx:44]: Tried it and there's still a bit of a flash. Not sure how to take that away.

**Pull request merged**